package/entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+#########################################################################################################################################
+# DISCLAIMER																																																														#
+# Copied from https://github.com/moby/moby/blob/ed89041433a031cafc0a0f19cfe573c31688d377/hack/dind#L28-L37															#
+# Permission granted by Akihiro Suda <akihiro.suda.cz@hco.ntt.co.jp> (https://github.com/rancher/k3d/issues/493#issuecomment-827405962)	#
+# Moby License Apache 2.0: https://github.com/moby/moby/blob/ed89041433a031cafc0a0f19cfe573c31688d377/LICENSE														#
+#########################################################################################################################################
+if [ -f /sys/fs/cgroup/cgroup.controllers ]; then
+	# move the init process (PID 1) from the root group to the /init group,
+  # otherwise writing subtree_control fails with EBUSY.
+  mkdir -p /sys/fs/cgroup/init
+  echo 1 > /sys/fs/cgroup/init/cgroup.procs
```

Comment:
Not just 1 : https://github.com/moby/moby/pull/42331

Reply:
Aaah, the `sed: write error` that that PR is referring to (or rather the referenced issue) is exactly what was bugging me the whole day.
Thanks for this! :smile:

Reply:
With only `1` there, it wouldn't work with `--init`, and other edge cases

Reply:
```suggestion
  xargs -rn1 < /sys/fs/cgroup/cgroup.procs > /sys/fs/cgroup/init/cgroup.procs || :
```